ui/src/assets/locales/en/translation.json

```diff
@@ -96,6 +96,10 @@
               "label": "Gateway",
               "route": "New Route",
               "consumer": "New Consumer"
+            },
+            "uibuilder": {
+              "label": "UI Builder",
+              "page": "New Page"
```

Comment:
We have one unused item here. Maybe we make `"New Page"` the `label` and delete the other one?